assemblies/assembly-configuring-postgresql-databases.adoc

```diff
@@ -0,0 +1,27 @@
+[id='assembly-configuring-postgresql-databases_{context}']
+= Configuring external PostgreSQL databases
+
+As an administrator, you can configure and use external PostgreSQL databases in {product}. Based on your needs, you can use a PostgreSQL certificate file to configure an external PostgreSQL instance using the Operator or Helm Chart.
+
+[NOTE]
+====
+{product-very-short} supports only configuring PostgreSQL databases. You are responsible for maintenance activities, such as backing up your data or configuring high availability (HA) for the configured PostgreSQL databases.
+
+Also, configuring PostgreSQL databases by using the {product} Operator is not intended for production use.
```

Comment:
As commented, "configuring PostgreSQL databases" is confusing IMHO, you configure DB connection anyway, we need to be more explicit like "configuring local (vs external)" term or use other term (like "installing", not ideal but more clear)